pkg/webui/console/components/webhook-form/index.js

```diff
@@ -363,12 +406,42 @@ export default class WebhookForm extends Component {
             sensitive
             code
           />
+          {this.state.hasAuthorization && (
+            <Notification content={m.hasAuthorization} small warning />
+          )}
+          <Form.Field
+            autoFocus
+            title={m.requestBasicAuth}
+            name="basic_auth.value"
```

Comment:
```suggestion
            name="headers"
            encode={...}
            decode={...}
```

I believe it would make things a lot easier if you use the encode decode functions here to marshal the different data shapes.